commands/inventory/add_agent_mongodb_exporter.go

```diff
@@ -16,25 +16,32 @@
 package inventory
 
 import (
+	"fmt"
+	"io/ioutil"
+
 	"github.com/percona/pmm/api/inventorypb/json/client"
 	"github.com/percona/pmm/api/inventorypb/json/client/agents"
+	"github.com/pkg/errors"
 
 	"github.com/percona/pmm-admin/commands"
 )
 
 var addAgentMongodbExporterResultT = commands.ParseTemplate(`
 MongoDB Exporter added.
-Agent ID              : {{ .Agent.AgentID }}
-PMM-Agent ID          : {{ .Agent.PMMAgentID }}
-Service ID            : {{ .Agent.ServiceID }}
-Username              : {{ .Agent.Username }}
-Listen port           : {{ .Agent.ListenPort }}
-TLS enabled           : {{ .Agent.TLS }}
-Skip TLS verification : {{ .Agent.TLSSkipVerify }}
-
-Status                : {{ .Agent.Status }}
-Disabled              : {{ .Agent.Disabled }}
-Custom labels         : {{ .Agent.CustomLabels }}
+Agent ID              		      : {{ .Agent.AgentID }}
+PMM-Agent ID                      : {{ .Agent.PMMAgentID }}
+Service ID                        : {{ .Agent.ServiceID }}
+Username                          : {{ .Agent.Username }}
+Listen port                       : {{ .Agent.ListenPort }}
+TLS enabled                       : {{ .Agent.TLS }}
+Skip TLS verification             : {{ .Agent.TLSSkipVerify }}
+TLS certificate key file          : {{ .Agent.TLSCertificateKeyFile }}
+TLS certificate key file password : {{ .Agent.TLSCertificateKeyFilePassword }}
```

Comment:
We definetily should not show certificate key file password and probably we don't want to show key file content.

Reply:
New fields removed from output. Also new fields was added to qan-mongodb-profiler command.